modules/install-creating-aws-local-zones-subnet.adoc

```diff
@@ -0,0 +1,77 @@
+:_content-type: PROCEDURE
+[id="install-creating-aws-local-zones-subnet"]
+=== Creating an AWS Local Zone subnet
+
+To create a Local Zone subnet, choose a zone group to enable and opt it in. Then, create the subnet.  
+
+.Procedure
+
+. To opt the zone group in, from a command line, enter the following command:
++
+[source,terminal]
+----
+$ aws ec2 modify-availability-zone-group \
+    --region ${AWS_REGION} \
+    --group-name ${ZONE_GROUP_NAME} \
+    --opt-in-status opted-in
+----
+
+// TODO: GitHub
+// TODO: Review yaml--is this only us-west-2-lax-la?
+. Create the stack for the Local Zone subnet `us-west-2-lax-1a`. For example:
++
+[source,terminal]
+----
+INSTALLER_URL="https://raw.githubusercontent.com/openshift/installer/master"
+TPL_URL="${INSTALLER_URL}/upi/aws/cloudformation/01.99_net_local-zone.yaml"
+
+aws cloudformation create-stack \
+    --region ${AWS_REGION} \
+    --stack-name ${SUBNET_NAME} \
+    --template-body ${TPL_URL} \
+    --parameters \
+        ParameterKey=VpcId,ParameterValue=${VPC_ID} \
+        ParameterKey=ZoneName,ParameterValue=${ZONE_NAME} \
+        ParameterKey=SubnetName,ParameterValue=${SUBNET_NAME} \
+        ParameterKey=PublicSubnetCidr,ParameterValue=${SUBNET_CIDR} \
+        ParameterKey=PublicRouteTableId,ParameterValue=${PUBLIC_RTB_ID}
+----
+
+. Enter the following command to wait for the stack creation to finish:
++
+[source,terminal]
+----
+aws cloudformation wait stack-create-complete \
+  --region ${AWS_REGION} \
+  --stack-name ${SUBNET_NAME}
+----
+// TODO: jq
+- Export the Local Zone subnet ID and append it to the subnet list. For example:
++
+[source,terminal]
+----
+export SUBNET_ID=$(aws cloudformation describe-stacks \
+  --region ${AWS_REGION} \
+  --stack-name ${SUBNET_NAME} \
+  | jq -r '.Stacks[0].Outputs[] | select(.OutputKey=="PublicSubnetIds").OutputValue' )
```

Comment:
`jq` was removed from the installer PR  https://github.com/openshift/installer/pull/6993

```suggestion
export SUBNET_ID=$(aws cloudformation describe-stacks \
  --region ${AWS_REGION} \
  --stack-name ${SUBNET_NAME} \
  --query 'Stacks[0].Outputs[?OutputKey==`PublicSubnetIds`].OutputValue' --output text)
```